plugin/src/main/groovy/com/novoda/buildproperties/Entries.groovy

```diff
@@ -7,4 +7,6 @@ interface Entries {
     Entry getAt(String key)
 
     Enumeration<String> getKeys()
+
+    Map<String, Entry> asMap()
```

Comment:
there is actually no reason for `Entries` to stay as interface, so I think that using an abstract class here to reduce the duplication is totally acceptable

Reply:
I just wonder whether this should be also `final` (given it's a convenience method built on the most generic `Entries` contract) 🤔